### PR TITLE
Handle requests for ephemeral_storage metrics

### LINKED ISSUE #50: The memory metrics are returned also for storage
Due to a coding error, the memory metrics are returned for **both** types.

Unfortunately the _real_ metrics are always zero, but that's another story...

### PR DESCRIPTION
**Pull request by afbjorklund**
Report any other non-existing metrics as NULL

Also add some database tests, for "storage"

Closes #50

### CODE REVIEW

File: pkg/api/dashboard/dashboard.go [hunk 1]
```diff
@@ -109,10 +109,12 @@ func getRows(db *sql.DB, table string, metricName string, selector ResourceSelec
 	orderBy := []string{"name", "time"}
 	if metricName == "cpu" {
 		query = "select sum(cpu), name, uid, time from %s "
-	} else {
-		//default to metricName == "memory/usage"
-		// metricName = "memory"
+	} else if metricName == "memory" {
 		query = "select sum(memory), name, uid, time from %s "
+	} else if metricName == "ephemeral_storage" {
```
Comment (ya-makariy) [inline pkg/api/dashboard/dashboard.go:114]: It seems to me that the metricName would be more appropriate - "storage"

Comment (afbjorklund) [reply pkg/api/dashboard/dashboard.go:114]: That is a different metric

        stmt, err := tx.Prepare("insert into nodes(uid, name, cpu, memory, storage, time) values(?, ?, ?, ?, ?, datetime('now'))")

                _, err = stmt.Exec(v.UID, v.Name, v.Usage.Cpu().MilliValue(), v.Usage.Memory().MilliValue()/1000, v.Usage.StorageEphemeral().MilliValue()/1000)

The database column could be renamed, but I don't think anyone cares.

Comment (ya-makariy) [reply pkg/api/dashboard/dashboard.go:114]: You are right, in my comment relied just on the title of the column

Comment (afbjorklund) [reply pkg/api/dashboard/dashboard.go:114]: as far as I know, `Storage` is _only_ used for persistent volumes - not for nodes and pods

https://github.com/kubernetes-retired/heapster/blob/master/docs/storage-schema.md

**Pull request closed without merging**